internal/commands/response_test.go

```diff
@@ -137,6 +137,19 @@ func TestResponse(t *testing.T) {
 			assert.Contains(t, err.Error(), "not implemented")
 		}
 	})
+
+	t.Run("filterBodyPrefix", func(t *testing.T) {
+		rtemp := Response{
+			body: []byte("No way, no how"),
+		}
+		if assert.True(t, rtemp.Ok()) {
+			r2 := rtemp.FilterBodyPrefix("No way")
+			assert.True(t, r2.Ok())
+			assert.Equal(t, []byte(""), r2.body)
+			err := r2.NoBody().End()
+			assert.NoError(t, err)
+		}
+	})
```

Comment:
Should there be an additional test that validates the negative case where the body is not filtered? Or is that somehow implicit from the above tests?

Reply:
Sure. I can add an explicit case for that.

Reply:
All set.